components/ws-manager-bridge/ee/src/bridge.ts

```diff
@@ -89,6 +89,9 @@ export class WorkspaceManagerBridgeEE extends WorkspaceManagerBridge {
                     prebuild.error = status.conditions!.headlessTaskFailed;
                     prebuild.snapshot = status.conditions!.snapshot;
                     headlessUpdateType = HeadlessWorkspaceEventType.FinishedButFailed;
+                } else if (!status.conditions!.snapshot) {
+                    prebuild.state = "failed";
```

Comment:
Suggestion: Here you could also set `prebuild.error` to something like "Prebuild does not have a snapshot".

Reply:
That is a symptom not the actual error. I've discussed with Gero that we will also relay overwriting actual errors if they exist (in case of multiple stop messages).